test/stage2/x86_64.zig

```diff
@@ -1700,6 +1700,36 @@ pub fn addCases(ctx: *TestContext) !void {
                 \\    if (!ok) unreachable;
                 \\}
             , "");
+            case.addCompareOutput(
```

Comment:
why not make these behavior tests instead? the intent of these tests is to verify incremental compilation, but it doesn't look like this exercises any new incremental compilation code; it only tests behavior.

Reply:
That's a great suggestion, thanks!

> > According to [Wikipedia](https://en.wikipedia.org/wiki/Instruction_selection), instruction selection is not really what the Emit phase is doing. This might be confusing to contributors which expect to find the generally accepted definition of instruction selection.
> 
> This prompts the question, should "Codegen" be renamed to "ISel"?

How about we bring it up at next stage2 meetup and discuss?

Reply:
I might land this as is and then do a sweep for any tests that should really end up in behavior test suite. Either way, why did I not think of it before - thanks again Andrew!